### PR TITLE
I18n/ru/core concepts/client side scripts and middlewares

### PR DESCRIPTION
**Pull request by RolginRoman**
#### What kind of changes does this PR include?

- Translated content
- https://github.com/withastro/docs/blob/main/src/content/docs/en/guides/middleware.mdx
- https://github.com/withastro/docs/blob/main/src/content/docs/en/guides/client-side-scripts.mdx

#### Description

- RU Tranlations for these pages

### CODE REVIEW

File: src/content/docs/ru/guides/middleware.mdx [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+<ul>
+    {orders.map(order => {
+        return <li>{/* любые действия с объектом `order` */}</li>;
+    })}
+</ul>
+```
+
+### Пример: цензурирование персональных данных
+
+Пример ниже использует middleware для того, чтобы заменить персональные дынные на странице ("ПД") словосочетанием "Совершенно секретно":
```
Comment (sobresaliente) [inline src/content/docs/ru/guides/middleware.mdx:118]: typo: should be 'данные' instead of 'дынные'

**Review by sobresaliente — COMMENTED** [review]
Hi! I've read through the pr and found small typos. Could you fix them, please?

File: src/content/docs/ru/guides/client-side-scripts.mdx [hunk 2]
```diff
@@ -0,0 +1,235 @@
...
+  }
+
+  // Оповещаем браузер, о том, что существует реализация для `astro-heart` тега.
+  customElements.define('astro-heart', AstroHeart);
+</script>
+```
+
+Есть 2 преимущества использования кастомных элементов здесь:
+
+1. Вместо того, чтобы искать элементы через `document.querySelector()`, здесь можно использовать `this.querySelector()`, который ищет только внутри текущего экземляра элемента. Это позволяет упростить работу с дочерними элементами внутри элемента.
```
[the remark this answers is not in the document]
Comment (RolginRoman) [reply src/content/docs/ru/guides/client-side-scripts.mdx]: @sobresaliente thanks a lot. fixed ✅

File: src/content/docs/ru/guides/middleware.mdx (hunk 1 above)
Comment (RolginRoman) [reply src/content/docs/ru/guides/middleware.mdx]: fixed ✅

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Thanks for the great work peeps 🚀

**Pull request merged**